base_jsonify/README.rst

```diff
@@ -79,6 +79,95 @@ you can pass a callable or the name of a method on the model:
 Also the module provide a method "get_json_parser" on the ir.exports object
 that generate a parser from an ir.exports configuration.
 
+Further features are available for advanced uses.
+It defines a simple "resolver" model that has a "python_code" field and an eval
+function so that arbitrary functions can be configured to transform fields,
+or process the resulting dictionary.
+It is also to specify a lang to extract the translation of any given field.
+
+To use these features, a full parser follows the following structure:
+
+.. code-block:: python
+
+    parser = {
+        "resolver": 3,
+        "language_agnostic": True,
+        "langs": {
+            False: [
+                {'name': 'description'},
+                {'name': 'number', 'resolver': 5},
+                ({'name': 'partner_id', 'alias': 'partners'}, [{'name': 'display_name'}])
```

Comment:
I'd take the change to rename `alias` to `target`, which is more explicit IMO.

Reply:
Thank you for the detailed review, I'll fix all these soon.